purchases/src/test/java/com/revenuecat/purchases/PurchasesCommonTest.kt

```diff
@@ -1506,7 +1320,7 @@ internal class PurchasesTest: BasePurchasesTest() {
         mockCacheStale(customerInfoStale = true)
         mockSynchronizeSubscriberAttributesForAllUsers()
         mockOfferingsManagerAppForeground()
-        Purchases.sharedInstance.onAppForegrounded()
+        Purchases.sharedInstance.purchasesOrchestrator.onAppForegrounded()
```

Comment:
I went back to using the purchaseOrchestrator for these... the one in `Purchases` is deprecated in the `defaults` flavor and it doesn't exist in the `customEntitlementComputation` flavor, so this allowed me to share the test between both flavors and use non-deprecated functions.